testcli.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	appsv1 "github.com/openshift/client-go/apps/clientset/versioned/typed/apps/v1"
+	projectsv1 "github.com/openshift/client-go/project/clientset/versioned/typed/project/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	//"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	kubeconfig = flag.String("kubeconfig", "", "absolute path to kubeconfig")
+)
+
+func doubleReplicas(appClientset *appsv1.AppsV1Client, namespace string) error {
+	//doubles the number of replicas in all Deployment Configs in a namespace
+
+	dcList, err := appClientset.DeploymentConfigs(namespace).List(metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+
+	for _, dc := range dcList.Items {
+		replicaCount := dc.Spec.Replicas
+		fmt.Printf("Deployment Config: %s old replicas: %d new replicas: %d\n", dc.ObjectMeta.Name, replicaCount, replicaCount*2)
+		dc.Spec.Replicas = replicaCount * 2
+		_, err := appClientset.DeploymentConfigs(namespace).Update(&dc)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func main() {
+
```

Comment:
kill this newline. never leave lines right after or just before the function ends